frontend/src/Pages/MeetTheTeamPage/MeetTheTeamPage.jsx

```diff
@@ -0,0 +1,233 @@
+  import adwaitAvatar from './../../../assets/TeamAvatars/Adwait.png';
+  import sanayAvatar from './../../../assets/TeamAvatars/Sanay.png';
+  import mirandaAvatar from './../../../assets/TeamAvatars/Miranda.png';
+  import johnAvatar from '../../../assets/TeamAvatars/John.png';
+  import cjImage from './../../../assets/TeamPics/cj image.jpg';
+  import graceImage from './../../../assets/TeamPics/grace image.jpg';
+  import hemanImage from './../../../assets/TeamPics/heman image.jpg';
+  import amandaImage from './../../../assets/TeamPics/amanda image.jpeg';
+  import cooperImage from './../../../assets/TeamPics/cooper image.jpg';
+
+  const MeetTheTeamPage = () => {
+    const teamMembersFirstRow = [
+      {
+        name: 'Maryellen Marino',
+        role: 'President',
+        image: maryellenImage,
+        avatar: maryellenAvatar,
+      },
+      {
+        name: 'Raven Levitt',
+        role: 'Vice President',
+        image: ravenImage,
+        avatar: ravenAvatar,
+      },
+      {
+        name: 'John Kloepfer',
+        role: 'Director of Technology',
+        image: johnImage,
+        avatar: johnAvatar,
+      },
+    ];
+  
+    const teamMembersSecondRow = [
+      {
+        name: 'Xenia Khusid',
+        role: 'Director of Logistics',
+        image: xeniaImage,
+        avatar: xeniaAvatar,
+      },
+      {
+        name: 'Adwait Naware',
+        role: 'Director of Finance',
+        image: adwaitImage,
+        avatar: adwaitAvatar,
+      },
+      {
+        name: 'Sanay Tralshawala',
+        role: 'Director of Sponsorship',
+        image: sanayImage,
+        avatar: sanayAvatar,
+      },
+      {
+        name: 'Miranda Zheng',
+        role: 'Director of Marketing',
+        image: mirandaImage,
+        avatar: mirandaAvatar,
+      },
+    ];
+
+    const jrTeamMembersFirstRow = [
+      {
+        name: 'CJ Marino',
+        role: 'Junior Director of Logistics',
+        image: cjImage,
+      },
+      {
+        name: 'Grace Hui',
+        role: 'Junior Director of Finance',
+        image: graceImage,
+      },
+    ];
+
+    const jrTeamMembersSecondRow = [
+      {
+        name: 'Heman Kolla',
+        role: 'Junior Director of Sponsorship',
+        image: hemanImage,
+      },
+      {
+        name: 'Amanda Ruan',
+        role: 'Junior Director of Marketing',
+        image: amandaImage,
+      },
+      {
+        name: 'Cooper Werner',
+        role: 'Junior Director of Technology',
+        image: cooperImage,
+      },
+    ];
+
+    return (
+      <div id="members" className="MeetTheTeamPage centeredText">
+        <h1 id="meet-our-team-header" style={{ fontFamily: 'Mokoto', fontSize: '46px' }}>
+          Meet Our Team!
+        </h1>
+
+        {/* Directors: */}
+
+        <div className="team-members">
+          <h1 className="director-jrDirector-header" style={{ padding: '0 7px' }}>
+            Meet The <a style={{ color: '#910307' }}>Directors</a>
```

Comment:
Is the color for the Directors and Junior Directors supposed to be two different colors? Is ok if they are just wondering.